### PR TITLE
Feature/dapp 153

### PR DESCRIPTION
**Pull request by letbueno**
# Description
- Forms to payment with credit card

## Does this pull request close any open issues?
- Put here the issue that must be closed with this PR 
- eg: closes #111

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] It is urgent
- [ ] <span style="color:#d44037">Dangerous change</span>

### CODE REVIEW

File: src/pages/promoters/SupportFundPage/CardSection/PaymentInformationSection/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "1234567890123456",
+    );
+
+    userEvent.type(screen.getByPlaceholderText("Name on card"), "User Test");
+
+    userEvent.type(screen.getByPlaceholderText("Expiration"), "1220");
+
+    userEvent.type(
+      screen.getByPlaceholderText("Card verification code"),
+      "123",
```
Comment (NickNish09) [inline src/pages/promoters/SupportFundPage/CardSection/PaymentInformationSection/index.test.tsx:30]: same thing as before

File: src/components/atomics/inputs/InputAutoComplete/index.tsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ function InputAutoComplete({
         placeholder={placeholder}
         aria-label={placeholder}
         name={name}
+        required
```
Comment (NickNish09) [inline src/components/atomics/inputs/InputAutoComplete/index.tsx:58]: maybe we could pass it as props

File: src/types/entities/CreditCardPayment.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export default interface CreditCardPayment {
+  country: string;
+  state: string;
+  city: string;
+  tax_id: string;
+  offer_id: number;
```
Comment (NickNish09) [inline src/types/entities/CreditCardPayment.ts:6]: same idea of the camel case thing, `taxId`, `offerId`

File: src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.test.tsx [hunk 4]
```diff
@@ -8,4 +10,15 @@ describe("BillingInformationSection", () => {
...
+
+  it("should fill billing information form", () => {
+    renderComponent(<BillingInformationSection />);
+
+    userEvent.type(screen.getByPlaceholderText("Country"), "Brazil");
+    userEvent.type(screen.getByPlaceholderText("City"), "São Paulo");
+
+    userEvent.type(screen.getByPlaceholderText("State"), "SP");
+
+    userEvent.type(screen.getByPlaceholderText("Tax ID"), "00000000000");
```
[the remark this answers is not in the document]
Comment (letbueno) [reply src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.test.tsx:22]: yess, this is will bring more security to the test

File: src/components/atomics/inputs/InputAutoComplete/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (letbueno) [reply src/components/atomics/inputs/InputAutoComplete/index.tsx:58]: i think i don't understand, like what?

[the remark this answers is not in the document]
Comment (NickNish09) [reply src/components/atomics/inputs/InputAutoComplete/index.tsx:58]: like passing to InputAutoComplete a prop, among the others that are already in this component
```
required: boolean;
```

and pass it down to this line.

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**